### PR TITLE
Surf remaining data

### PR DESCRIPTION
**Pull request by fredrikleknes**
Added Riser count and Template count to Surf

### CODE REVIEW

File: frontend/src/Views/CaseView.tsx [hunk 1]
```diff
@@ -43,6 +43,8 @@ function CaseView() {
     const [producerCount, setProducerCount] = useState<number>()
     const [gasInjectorCount, setGasInjectorCount] = useState<number>()
     const [waterInjectorCount, setWaterInjectorCount] = useState<number>()
+    const [riserCount, setRicerCount] = useState<number>()
```
Comment (DanielBohme) [inline frontend/src/Views/CaseView.tsx:46]: setRicerCount should be setRiserCount

File: frontend/src/Views/CaseView.tsx [hunk 2]
```diff
@@ -66,6 +68,8 @@ function CaseView() {
             setProducerCount(caseResult?.producerCount)
             setGasInjectorCount(caseResult?.gasInjectorCount)
             setWaterInjectorCount(caseResult?.waterInjectorCount)
+            setRicerCount(caseResult?.riserCount)
```
Comment (DanielBohme) [inline frontend/src/Views/CaseView.tsx:71]: should be setRiserCount

File: frontend/src/Views/CaseView.tsx [hunk 3]
```diff
@@ -171,6 +177,22 @@ function CaseView() {
                         label="Rig mob demob"
                     />
                 </Wrapper>
+                <Wrapper>
+                    <NumberInput
+                        setValue={setRicerCount}
```
Comment (DanielBohme) [inline frontend/src/Views/CaseView.tsx:182]: should be setRiserCount

Comment (fredrikleknes) [reply frontend/src/Views/CaseView.tsx]: Fixed

File: frontend/src/Views/CaseView.tsx (hunk 2 above)
Comment (fredrikleknes) [reply frontend/src/Views/CaseView.tsx]: Fixed

**Review by DanielBohme — DISMISSED** [review]
LGTM

**Review by petterwildhagen — APPROVED** [review]
LGTM

**Pull request merged**